lib/jsxgettext.js

```diff
@@ -53,6 +53,9 @@ function getTranslatable(node, options) {
   if (options.keyword.indexOf(funcName) === -1)
     return false;
 
+  if (arg && funcName.substr(0, 1) === "n" && (isStrConcatExpr(arg) || isStringLiteral(arg)) && node.arguments[1] && (isStrConcatExpr(node.arguments[1]) || isStringLiteral(node.arguments[1])))
```

Comment:
I guess here we have 2 options:
- leave it this way and explain that any gettext alias starting with "n" will be treated as a plural function
- use a strict comparison so that only "ngettext" will work. In this case there would be no way of using custom plural function until someone implements some kind of way to specify that in the calls
I'm comfortable with both these solutions, so it's your call